ceph_cfg/tests/test_configparserceph.py

```diff
@@ -0,0 +1,35 @@
+from ceph_cfg.util_configparser import ConfigParserCeph as ConfigParser
+import io
+
+class Test_util_configparser(object):
+
+    def test_whitespace_none(self):
+        sample_config = """
+[mysqld]
+user2 = mysql
+"""
+        config = ConfigParser()
+        config.readfp(io.BytesIO(sample_config))
+        value = config.get("mysqld", "user2")
+        assert value == "mysql"
+
+
+    def test_whitespace(self):
+        sample_config = """
+[mysqld]
+user 2 = mysql
+"""
+        config = ConfigParser()
+        config.readfp(io.BytesIO(sample_config))
+        value = config.get("mysqld", "user_2")
+        assert value == "mysql"
+
+    def test_whitespace_two(self):
+        sample_config = """
```

Comment:
nit: for completeness sake we can probably add one more test case for 
```
[mysqlid]
user_2 = foo
``` 
and see that this also works